docs/docs/noir/concepts/data_types/structs.md

```diff
@@ -79,4 +81,16 @@ pub struct Animal {
     legs: Field,
     eyes: u8,
 }
+```
+
+The same applies to struct fields: by default they are private to the module they exist in,
+but they can be made `pub` or `pub(crate)`:
+
+```rust
+// This struct is now public
+pub struct Animal {
+    hands: Field,           // private to its module
+    pub(crate) legs: Field, // accessible from the entire crate
```

Comment:
Please also add a test or two for the `pub(crate)` case(s). (It may need to be a `test_programs` test.)

Reply:
Actually... it can't be done because these are warnings. Or we'd need a new category like `compile_success_no_warnings`. Should I do that?

Reply:
Will the warnings become errors in a later PR? If so, then please make a note to follow up (e.g. in the issue to make them errors). Otherwise, this seems important enough to add `compile_success_no_warnings`, but let's save that for another PR.